src/amp-story-player/amp-story-player-impl.js

```diff
@@ -169,6 +171,15 @@ export class AmpStoryPlayer {
       isSwipeX: null,
     };
 
+    /** @private {?Promise} */
+    this.currentStoryLoadPromise_ = null;
```

Comment:
Nit: I think this file will grow a lot, combining these 3 into 1 `currentStoryLoadDeferred_` might be worth it